content/source/docs/enterprise-beta/private/install.html.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation"
+sidebar_current: "docs-enterprise2-private-install"
+---
+
+# Private Terraform Enterprise Installation
+
+## Delivery
+
+The goal of this installation procedure is to set up a Terraform Enterprise
+cluster that is available on a DNS name that is accessed via HTTPS. This
+standard configuration package uses Terraform to create both the compute and data layer resources, and optionally uses Route53 to configure the DNS automatically.
+
+## Preflight
+
+### Dependencies
+
+Before setup begins, a few resources need to be provisioned. We consider these out of scope for the cluster provisioning because they depend on the user's environment.
+
+The following are required to complete installation:
+
+* **AWS IAM credentials** capable of creating new IAM roles configuring various services. We suggest you use an admin role for this. The credentials are only used for setup, during runtime only an assumed role is used.
+* **AWS VPC** containing at least 2 subnets. These will be used to launch the cluster into. Subnets do not need to be public, but they do need an internet gateway at present.
+* **SSH Key Pair** configured with AWS EC2. This will be used to configure support access to the cluster. This SSH key can be optionally removed from the instance once installation is complete.
+  * To create a new one: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html
```

Comment:
Change bare URL to a link, with text like `"To create a new one, [see Amazon's documentation of EC2 key pairs.](URL)"`